lib/Migration/Version035000Date20260529130000.php

```
@@ -0,0 +1,113 @@
+		$schema = $schemaClosure();
+		foreach (self::LEGACY_TABLES as $table) {
+			if ($schema->hasTable($table)) {
+				$schema->dropTable($table);
+				$this->dropped = true;
+			}
+		}
+
+		return $this->dropped ? $schema : null;
+	}
+
+	public function postSchemaChange(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$output'. (undefined)

(UnusedFormalParameter)